internal/services/cdn/cdn_frontdoor_route_disable_link_to_default_domain_resource.go

```diff
@@ -72,63 +71,63 @@ func resourceCdnFrontDoorRouteDisableLinkToDefaultDomainCreate(d *pluginsdk.Reso
 
 	customDomains := d.Get("cdn_frontdoor_custom_domain_ids").([]interface{})
 
-	routeId, err := parse.FrontDoorRouteID(d.Get("cdn_frontdoor_route_id").(string))
+	routeId, err := parse.FrontDoorRouteIDInsensitively(d.Get("cdn_frontdoor_route_id").(string))
```

Comment:
this is coming from the config which is validated and so should be being parsed case-sensitively:

```suggestion
	routeId, err := parse.FrontDoorRouteID(d.Get("cdn_frontdoor_route_id").(string))
```

Reply:
Fixed.